<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1413</width>
    <height>947</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thoth</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="titleFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="prev">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="Thoth.qrc">
          <normaloff>:/Thoth/Resources/icons/ic_chevron_left_black_36dp.png</normaloff>:/Thoth/Resources/icons/ic_chevron_left_black_36dp.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>No title</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="next">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="Thoth.qrc">
          <normaloff>:/Thoth/Resources/icons/ic_chevron_right_black_36dp.png</normaloff>:/Thoth/Resources/icons/ic_chevron_right_black_36dp.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="newLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>New</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="mainFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QSplitter" name="mainSplitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="SceneListWidget" name="sceneList" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
         </widget>
         <widget class="LabelListWidget" name="labelList" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="menuFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>24</number>
             </property>
             <property name="leftMargin">
              <number>13</number>
             </property>
             <property name="topMargin">
              <number>13</number>
             </property>
             <property name="rightMargin">
              <number>13</number>
             </property>
             <property name="bottomMargin">
              <number>13</number>
             </property>
             <item>
              <widget class="QToolButton" name="select">
               <property name="text">
                <string>Select</string>
               </property>
               <property name="icon">
                <iconset resource="Thoth.qrc">
                 <normaloff>:/Thoth/Resources/img/ic-near-me.png</normaloff>:/Thoth/Resources/img/ic-near-me.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="rect">
               <property name="text">
                <string>Rectangle</string>
               </property>
               <property name="icon">
                <iconset resource="Thoth.qrc">
                 <normaloff>:/Thoth/Resources/img/icon-retangle.png</normaloff>:/Thoth/Resources/img/icon-retangle.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="poly">
               <property name="text">
                <string>Polygon</string>
               </property>
               <property name="icon">
                <iconset resource="Thoth.qrc">
                 <normaloff>:/Thoth/Resources/img/icon-polygon.png</normaloff>:/Thoth/Resources/img/icon-polygon.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="merge">
               <property name="text">
                <string>Merge</string>
               </property>
               <property name="icon">
                <iconset resource="Thoth.qrc">
                 <normaloff>:/Thoth/Resources/img/icon-merge.png</normaloff>:/Thoth/Resources/img/icon-merge.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="buffer">
               <property name="statusTip">
                <string>Buffering</string>
               </property>
               <property name="whatsThis">
                <string>Expand selecte item</string>
               </property>
               <property name="text">
                <string>Buffer</string>
               </property>
               <property name="icon">
                <iconset resource="Thoth.qrc">
                 <normaloff>:/Thoth/Resources/img/icon-buffer.png</normaloff>:/Thoth/Resources/img/icon-buffer.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="sync">
               <property name="statusTip">
                <string>Sync.</string>
               </property>
               <property name="whatsThis">
                <string>Sync.</string>
               </property>
               <property name="text">
                <string>Sync.</string>
               </property>
               <property name="icon">
                <iconset resource="Thoth.qrc">
                 <normaloff>:/Thoth/Resources/img/icon-sync.png</normaloff>:/Thoth/Resources/img/icon-sync.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="osm">
               <property name="statusTip">
                <string>OSM filter</string>
               </property>
               <property name="whatsThis">
                <string>Show OSM Filter </string>
               </property>
               <property name="text">
                <string>OSM Filter</string>
               </property>
               <property name="icon">
                <iconset resource="Thoth.qrc">
                 <normaloff>:/Thoth/Resources/img/ic-map.png</normaloff>:/Thoth/Resources/img/ic-map.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="undo">
               <property name="statusTip">
                <string>Undo</string>
               </property>
               <property name="whatsThis">
                <string>Undo</string>
               </property>
               <property name="text">
                <string>Undo</string>
               </property>
               <property name="icon">
                <iconset resource="Thoth.qrc">
                 <normaloff>:/Thoth/Resources/icons/ic_undo_black_36dp.png</normaloff>:/Thoth/Resources/icons/ic_undo_black_36dp.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="currentPos">
               <property name="text">
                <string>X:0, Y:0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="settings">
               <property name="statusTip">
                <string>Undo</string>
               </property>
               <property name="whatsThis">
                <string>Undo</string>
               </property>
               <property name="text">
                <string>Undo</string>
               </property>
               <property name="icon">
                <iconset resource="Thoth.qrc">
                 <normaloff>:/Thoth/Resources/icons/outline_settings_black_36dp.png</normaloff>:/Thoth/Resources/icons/outline_settings_black_36dp.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="GraphicsWidget" name="map" native="true">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>400</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>400</width>
              <height>400</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="CategoryWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SceneListWidget</class>
   <extends>QWidget</extends>
   <header>SceneListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LabelListWidget</class>
   <extends>QWidget</extends>
   <header>LabelListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphicsWidget</class>
   <extends>QWidget</extends>
   <header>GraphicsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CategoryWidget</class>
   <extends>QWidget</extends>
   <header>CategoryWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Thoth.qrc"/>
 </resources>
 <connections/>
</ui>
